Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp

```diff
@@ -62,6 +64,19 @@ extern PropertyRNA* BlenderPyContext_blend_data;
 extern PropertyRNA* BlenderPyContext_scene;
 extern FunctionRNA* BlenderPyContext_evaluated_depsgraph_get;
 extern FunctionRNA* BlenderPyContext_depsgraph_update;
+extern PropertyRNA* BlenderPyContext_viewlayer;
```

Comment:
non blocking, this should be view_layer

Reply:
Good catch! I ended up removing this code